Libbeat panic when float32 value returned in metricset

- Version: 
libbeat 7.0.0-alpha1 [72de9eddd42a7c688693b54c587778653d5ffecd]

- Operating System: 
Linux

- Steps to Reproduce: 
```
func (m *MetricSet) Fetch() (common.MapStr, error) {
  return common.MapStr{
    "value": float32(1),
  }, nil
}
```

Whats happening:
```
panic: interface conversion: interface {} is float32, not float64

goroutine 38 [running]:
github.com/elastic/beats/libbeat/common.normalizeValue(0x26aa620, 0x3ada1a0, 0xc42025fc10, 0x3, 0xa, 0x253f9f0, 0x25353d2, 0x26e0ae0, 0x25340d3, 0x2790a40)
	/home/ixtiyor/projects/golang/src/github.com/elastic/beats/libbeat/common/event.go:156 +0x2125
github.com/elastic/beats/libbeat/common.normalizeMap(0xc420298c00, 0xc42025fc10, 0x2, 0xa, 0x0, 0x0, 0x0, 0x0)
	/home/ixtiyor/projects/golang/src/github.com/elastic/beats/libbeat/common/event.go:47 +0x1c0
github.com/elastic/beats/libbeat/common.normalizeValue(0x29d5300, 0xc420298c00, 0xc42025fc10, 0x2, 0xa, 0x26c2ae0, 0xc420152e60, 0x0, 0x0, 0x0)
	/home/ixtiyor/projects/golang/src/github.com/elastic/beats/libbeat/common/event.go:162 +0x148c
github.com/elastic/beats/libbeat/common.normalizeMap(0xc420299020, 0xc42025fc10, 0x1, 0xa, 0x0, 0x0, 0x0, 0x0)
	/home/ixtiyor/projects/golang/src/github.com/elastic/beats/libbeat/common/event.go:47 +0x1c0
github.com/elastic/beats/libbeat/common.normalizeValue(0x29d5300, 0xc420299020, 0xc42025fc10, 0x1, 0xa, 0x29d5300, 0xc420299200, 0x0, 0x0, 0x0)
	/home/ixtiyor/projects/golang/src/github.com/elastic/beats/libbeat/common/event.go:162 +0x148c
github.com/elastic/beats/libbeat/common.normalizeMap(0xc420298f90, 0xc42025fc10, 0x0, 0xa, 0x0, 0x0, 0x0, 0x0)
	/home/ixtiyor/projects/golang/src/github.com/elastic/beats/libbeat/common/event.go:47 +0x1c0
```

--- a/CHANGELOG.asciidoc
+++ b/CHANGELOG.asciidoc
@@ -59,7 +59,7 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Fix documentation links in README.md files. {pull}5710[5710]
 - Fix logstash output debug message. {pull}5799{5799]
 - Fix isolation of modules when merging local and global field settings. {issue}5795[5795]
-- Fix panic in normalizeValue, when published common.MapStr contains float32 type data {pull}[6129]
+- Fix panic when Events containing a float32 value are normalized. {pull}6129[6129]
 
 *Auditbeat*
 

--- a/CHANGELOG.asciidoc
+++ b/CHANGELOG.asciidoc
@@ -59,6 +59,7 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Fix documentation links in README.md files. {pull}5710[5710]
 - Fix logstash output debug message. {pull}5799{5799]
 - Fix isolation of modules when merging local and global field settings. {issue}5795[5795]
+- Fix panic in normalizeValue, when published common.MapStr contains float32 type data {pull}[6129]
 
 *Auditbeat*
 

--- a/libbeat/common/event.go
+++ b/libbeat/common/event.go
@@ -152,8 +152,10 @@ func normalizeValue(value interface{}, keys ...string) (interface{}, []error) {
 	case []int, []int8, []int16, []int32, []int64:
 	case uint, uint8, uint16, uint32, uint64:
 	case []uint, []uint8, []uint16, []uint32, []uint64:
-	case float32, float64:
+	case float64:
 		return Float(value.(float64)), nil
+	case float32:
+		return Float(value.(float32)), nil
 	case []float32, []float64:
 	case complex64, complex128:
 	case []complex64, []complex128:

--- a/libbeat/common/event_test.go
+++ b/libbeat/common/event_test.go
@@ -237,6 +237,23 @@ func TestNormalizeValue(t *testing.T) {
 
 		assert.Equal(t, test.out, out, "Test case %v", i)
 	}
+
+	var floatTests = []struct {
+		in  interface{}
+		out interface{}
+	}{
+		{float32(1), float64(1)},
+		{float64(1), float64(1)},
+	}
+
+	for i, test := range floatTests {
+		out, err := normalizeValue(test.in)
+		if err != nil {
+			t.Error(err)
+			continue
+		}
+		assert.InDelta(t, test.out, float64(out.(Float)), 0.000001, "(approximate) Test case %v", i)
+	}
 }
 
 func TestNormalizeMapError(t *testing.T) {

